GameServerCore/Content/Color.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace GameServerCore.Content
+{
+    public class Color
+    {
+        public byte R { get; }
+        public byte G { get; }
+        public byte B { get; }
+        public float A { get; }
+    }
```

Comment:
might want to make all of those either ``byte`` or ``float``

Reply:
4.2 packet uses only Alpha as float `public SetScreenTint(TeamId team, bool enable, float transitionTime, byte red, byte green, byte blue, float alpha)`, so Im not sure how to convert byte <-> float.

Reply:
To convert byte to float you can divide with 255 and to convert float to byte you can multiply with 255.

Reply:
Changed.